Description: enable bytemuck derive feature
 The bytemuck feature fails to compile because it uses derive(bytemuck::Pod,
 bytemuck::Zeroable) without enabling the required "derive" feature in the
 bytemuck dependency. This updates the dependency to include the derive
 feature when bytemuck is enabled, fixing compilation with --features bytemuck
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Bug: https://github.com/kurtlawrence/guid-create/issues/26
Last-Update: 2025-09-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,6 +48,7 @@
 [dependencies.bytemuck]
 version = "1"
 optional = true
+features = ["derive"]
 
 [dependencies.rand]
 version = "0.8"
